sdk/lib/drivers/sound/mmixer/mixer.c

```diff
@@ -836,18 +836,17 @@ MMixerInitialize(
             }
             else
             {
-                DPRINT1("Failed to enumerate device %lu\n", DeviceIndex);
+                DPRINT1("Failed to enumerate device %lu, Status %x\n", DeviceIndex, Status);
 
-                /* TODO cleanup */
-                return Status;
+                /* ignore error and continue */
             }
         }
         else
         {
             /* create a mixer data entry */
             Status = MMixerCreateMixerData(MixerContext, MixerList, DeviceIndex, DeviceName, hMixer, hKey);
-            if (Status != MM_STATUS_SUCCESS)
```

Comment:
Maybe you could still log an error (like the DPRINT1 above) when it fails, but then continue execution as you do now.

Reply:
OK, done. Thanks. Although I changed an error text a bit to correspond to the actual error.